ocaml/typing/printtyped.ml

```diff
@@ -222,7 +222,7 @@ let attributes i ppf l =
   ) l
 
 let jkind_annotation i ppf (jkind, _) =
-  line i ppf "%s" (Jkind.string_of_const jkind)
+  line i ppf "%a" Jkind.Const.format jkind
```

Comment:
It's possible that `Const.format` is the wrong thing here. We want this output to be pretty raw. If that function evolves to be too cooked, we should do something different here.

Reply:
I've updated it to be more explicit